### DISCUSSION TITLE
July 22 Release - New themes 🎨, bulk upload users, new components panel and UX, and more.

### DISCUSSION THREAD

**Discussion by joebudi** [Budibase/budibase#7191] [Announcements]
Today we are launching the Budibase July release . This release introduces several improvements and quality-of-life updates that will improve the developer experience and spark joy in users. It also introduces two new builder themes; Nord 🔵 and Midnight ⚫.

### Features & Improvements ✨

**Introducing the Nord 🔵 and Midnight themes ⚫**
We have introduced two beautiful new themes to the Budibase platform - Nord and Midnight. You can use them by changing your theme. To access go to: Home screen  > Theming.
![nord theme](https://user-images.githubusercontent.com/49767913/183847615-d5801a9c-86b7-4eb1-83bf-a3ce416e0201.jpeg)

**Deactivate automations after several failures 🤖**
We’ve added a feature that will automatically stop your CRON automations and notify you in the automation log section if something is wrong.

**Bulk-upload users 👥**
You can now bulk add users to your Budibase installation in several different ways:
- Bulk-upload users (including their roles) to Budibase using a CSV
- Add many users at once as part of the invite UI

**New components panel and experience 🍱**
You asked - we listened. We’ve shifted the component panel to the right and added a sweet animation, making it easier than ever to add components. This also allows you to have the component tree and component panel open simultaneously. We’ve also added clearer CTAs for adding components and screens so that it’s much more obvious what to click when designing the builder.

**New access level color scale 🎨**
The new access control scale uses a color scale to make it easier for users to see the hierarchy of Budibase access roles.

**Instant Budibase setup through environment variables 🧑‍💻**
You can now set up a fresh Budibase installation with no manual user input by supplying the BB_ADMIN_USER_EMAIL and BB_ADMIN_USER_PASSWORD environment variables. This is supported in both the single image and docker-compose setups.

---

### Quality of life improvements 🤗

**Make full cards clickable 🖱️**
We added an option to make full card components clickable and to add handlers to them.

**Table block - more filters 🗂️**
The table block was previously limited to three filters. We’ve increased this limit to provide more filtering options for those using the table block.

**New user types  👥**
There have always been three standard types of Budibase users; App user, developer, and admin. We’ve clarified this distinction in the builder to improve UX and onboarding.

**Linode support  🌐**
Budibase is now on the [Linode marketplace](https://www.linode.com/marketplace/apps/budibase/budibase/), allowing you to deploy Budibase to Linode with one click.

---

### Bug Fixes 🐛
Remove user table as an option for automation triggers and create row action #6702
Error publishing app - status 500 when using CRON Trigger #6501
Selected form step not showing #6729
Background outside the Screen Slot no longer changeable. #6657
Format column date in a table does not directly work #6433
bbapps High ram usage #6580
Usage quota doc and user documents have an overlap in ID #5803
Single Image healthcheck fails if REDIS_PASSWORD is not set #6795
Update public api range filter's format #6434
Can't filter on query results when filtering for an integer column #6633
Cant save email configuration - "createdAt" is not allowed #6681
Filter drawer points to the wrong table when multiple table blocks / data providers are on screen #6905
Fix for SQL column removal (external to Budibase) #7028

---

**Coming in the next release 🔮**
- We're skipping the August release and preparing for a larger ~~release~~ launch. Stay tuned 👀

---

**Thank you to our community 🙏**
Thank you to our community. We are ever so thankful for your feedback and assistance. We've recently surpassed 14,400 Github stars, and 35,000 companies are now using Budibase, which is a great triumph. If you have not starred us, please do - it means a lot. Also, a huge shout to @poirazis and @R2bEEaton, who've helped other community members and provided awesome feedback 💙

**Comment by validlogin** [comment]
Very curious as the what the new release, err.. launch will be 👀 any hints for us? :)
